src/useInterval/useInterval.stories.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { bind, computed, defineComponent, propType, ref } from '@muban/muban';
+import type { Story } from '@muban/storybook/types-6-0';
+import { html } from '@muban/template';
+import { useInterval } from './useInterval';
+import { useStorybookLog } from '../hooks/useStorybookLog';
+
+export default {
+  title: 'useInterval',
+};
+
+type DemoStoryProps = { startImmediate?: boolean; interval?: number };
+
+export const Demo: Story<DemoStoryProps> = () => ({
+  component: defineComponent({
+    name: 'story',
+    props: {
+      startImmediate: propType.boolean.defaultValue(false),
+      interval: propType.number,
+    },
+    refs: {
+      label: 'label',
+      startButton: 'start-button',
+      stopButton: 'stop-button',
+    },
+    setup({ refs, props }) {
+      const [logBinding, log] = useStorybookLog(refs.label);
+      const isIntervalRunning = ref(false);
```

Comment:
Should we perhaps manage and expose this state as part of the hook? Could be useful.